run/idp-sql/app.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
What's this file used for? 

Should we include a header in it?

Reply:
This is for the Cloud Run button to build the sample correctly. Unfortunately I can only specify in JSON.